api/api_controller.go

```diff
@@ -2297,3 +2303,11 @@ func (c *Controller) ImportFromS3InventoryHandler() repositories.ImportFromS3Inv
 		})
 	})
 }
+
+func (c *Controller) ConfigGetConfigHandler() configop.GetConfigHandler {
+	return configop.GetConfigHandlerFunc(func(params configop.GetConfigParams, user *models.User) middleware.Responder {
+		return configop.NewGetConfigOK().WithPayload(&models.Config{
```

Comment:
This one is easy to miss: Please add a call to `c.setupRequest` here for authorization.  It's OK to add a new auth type if there is no suitable one already.  Have a look all over this file, every handler func here has one.

Reply:
I also wrote it in the slack help channel but I'll write here as well for the slight chance you'll read it first. 
I don't understand how to test it and would appreciate your guidance on this